src/System.Private.ServiceModel/tools/test/Bridge.Build.Tasks/Bridge.Build.Tasks/ReleaseBridgeResourcesTask.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Bridge.Build.Tasks
+{
+    // This task will invoke the Bridge to ask it to release all its
+    // resources.
+    public class ReleaseBridgeResourcesTask : Task
+    {
+        [Required]
+        public string BridgeHost { get; set; }
+
+        [Required]
+        public string BridgePort { get; set; }
+
+        public override bool Execute()
+        {
+            int port = 0;
+            if (!int.TryParse(BridgePort, out port))
+            {
+                Log.LogError("The BridgePort value '{0}' is not a valid integer.", BridgePort);
+                return false;
+            }
+
+            string bridgeAddress = string.Format("http://{0}:{1}", BridgeHost, BridgePort);
+
+            HttpClient httpClient = new HttpClient();
+            bool bridgeIsRunning = false;
+
+            try
+            {
+                // First, try pinging it.  If it is not currently running, we can safely
+                // ignore errors attempting to release its resource folder.
+                var response = httpClient.GetAsync(bridgeAddress).GetAwaiter().GetResult();
+                if (!response.IsSuccessStatusCode)
+                {
+                    string reason = String.Format("The ping GET request to the Bridge returned unexpected status code='{0}', reason='{1}'",
+                            response.StatusCode, response.ReasonPhrase);
+                    if (response.Content != null)
+                    {
+                        string contentAsString = response.Content.ReadAsStringAsync().GetAwaiter().GetResult();
```

Comment:
I would limit how long the string is that you output. A server could have a large error page. Maybe limit to 1000 characters?